docs/how/updating-datahub.md

```diff
@@ -19,6 +19,17 @@ This file documents any backwards-incompatible changes in DataHub and assists pe
 ## Next
 
 ### Breaking Changes
+- #9857 (#10773) `lower` method was removed from `get_db_name` of `SQLAlchemySource` class. This change will affect the urns of all related to `SQLAlchemySource` entities.
+  
+  Old `urn`, where `data_base_name` is `Some_Database`:
+  ```
+  - urn:li:dataset:(urn:li:dataPlatform:paltform,some_database.schema.entitie,ENV)
```

Comment:
```suggestion
  - urn:li:dataJob:(urn:li:dataFlow:(mssql,demodata.Foo.stored_procedures,PROD),Proc.With.SpecialChar)
```

Reply:
Let's change the example to be of dataJob as only dataJob and container URNs have changed and not that of dataset.

Reply:
Fixed.